React-frontend/src/components/Admin/AddMemberForm.js

```diff
@@ -20,6 +20,8 @@ import { useDispatch, useSelector } from 'react-redux';
 import { useHistory } from 'react-router-dom';
 import { addMember } from '../../store/actions/admin';
 import SelectItem from '../Utils/SelectItem';
+import { authentication } from '../../Firebase/firebase-config';
```

Comment:
AddMemberForm.js has already been updated in the previous PR no need to update the same again and again!